docs/registry.md

```diff
@@ -1,20 +1,27 @@
 # Configure Image Registry
 This document describes the method to configure the image registry for `containerd` for use with the `cri` plugin.
 
+NOTE: The configuration syntax used in this doc is in version 2 which is the
+recommended since containerd 1.13.0. If your configuration is still in version 1,
```

Comment:
1.13.0 -> 1.3

Reply:
done

Reply:
Seems not fixed yet. "containerd 1.13" was never released out.

Reply:
fixed

sorry about my stupid mistake...